cylc/flow/xtrigger_mgr.py

```diff
@@ -211,6 +211,11 @@ def __init__(
         # Signatures of active functions (waiting on callback).
         self.active: list = []
 
+        # Gather named wall-clocks closet to current time.
+        # (no need to check or spawn future clocks of tasks respectively).
+        self.wall_clock_labels: set = set()
+        self.wall_clock_spawns: list = []
```

Comment:
Could this variable be named better?  like `clock_spawn_next` or something?

Reply:
Yes, will change.. maybe to broaden `sequential_spawn_next`